src/modules/partition/gui/EncryptWidget.cpp

```diff
@@ -143,18 +146,24 @@ applyPixmap( QLabel* label, CalamaresUtils::ImageType pixmap )
 }
 
 void
-EncryptWidget::updateState()
+EncryptWidget::updateState( const bool notify )
```

Comment:
`const` value parameters always seem a bit sketchy to me: they're copies of something and their constness doesn't really matter (unless you're concerned that `notify` might be changed inside the function and the logic gets confused).

Reply:
That was my general reasoning, nothing should be changing it inside the function.